pyabc/external.py

```diff
@@ -9,6 +9,13 @@
     The rpy2 package needs to be installed to interface with the R language.
     Installation of rpy2 is optional if R support is not required.
     See also :ref:`installation of optional dependencies <install-optional>`.
+
+.. note::
+
+    Support of R via rpy2 is considered experimental. Currently, an rpy2
+    version <= 2.9.5 is required. Versions >= 3.0.0 do not work yet properly
```

Comment:
add this version checking already in setup.py ?
Maybe this is already there

Reply:
yes, in setup.py we currently require rpy2<=2.9.5 already. However, the user might have updated it somewhere else at some point, so the info here might be useful.